zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -640,6 +652,17 @@ private boolean canScanNode(StructuralNode node) {
             }
             return false;
         }
+        FilterResult filterResult = filterNode(node);
+        if (!filterResult.isFiltered()) {
```

Comment:
Am I missing something? If we're logging that something was filtered shouldn't isFiltered return true here?

Reply:
hmm.... Actually i was confused with this , i thought the otherway around ie isFiltered means passed through the filter successfully. i will do the changes.